experiments/experiments-api/src/main/java/com/duckduckgo/experiments/api/VariantConfig.kt

```diff
@@ -24,4 +24,5 @@ data class VariantConfig(
 
 data class VariantFilters(
     val locale: List<String>? = null,
+    val androidVersion: List<String>? = null,
```

Comment:
I don't get why we need nullable lists? a null list should rather be an empty list.
IIRC JSON (de)serialization returns empty list when not present

Reply:
Sure, I was duplicating the behaviour from locale filter but makes sense to make it not nullable as I don't see the case we want to return a null here instead an empty list